ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -39,6 +68,16 @@ Add OpenShift Groups To Data Science User Groups
     END
     Press Keys    None    ESC
 
+Open ODS Dashboard With Admin User
+    [Documentation]    Opens a browser and logs into ODS Dashboard with a user belonging to the rhods-admins group
+    Launch Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+
+Open ODS Dashboard With Non Admin User
+    [Documentation]    Opens a browser and logs into ODS Dashboard with a user belonging to the rhods-users group
+    Launch Dashboard    ${TEST_USER_3.USERNAME}    ${TEST_USER_3.PASSWORD}    ${TEST_USER_3.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+
```

Comment:
these don't look like specific for Dashboard settings page. Could you pls move to a more generic common file?